bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/dto/AlternativePostalAddressDto.kt

```diff
@@ -27,7 +27,10 @@ import org.eclipse.tractusx.bpdm.common.model.DeliveryServiceType
 import org.eclipse.tractusx.bpdm.common.service.DataClassUnwrappedJsonDeserializer
 
 @JsonDeserialize(using = DataClassUnwrappedJsonDeserializer::class)
-@Schema(description = PostalAddressDescription.headerAlternative)
+@Schema(
+    description = PostalAddressDescription.headerAlternative,
+    requiredProperties = ["country", "city", "deliveryServiceType", "deliveryServiceNumber"]
```

Comment:
I don't know, they are definitely needed to show the required-asterisks in the Swagger-UI.
In the Pool I see no fields marked as required at first glance.
But I agree, I really don't like this redundant requiredProperties annotation.
It would be really cool to get it derived from the field nullability.
I suggest we have a deeper look into it in another ticket.